package.json

```diff
@@ -47,17 +47,18 @@
     "babel-cli": "6.10.1",
     "babel-plugin-transform-es2015-arrow-functions": "6.8.0",
     "babel-plugin-transform-es2015-literals": "6.8.0",
-    "docpress": "0.6.11",
-    "eslint": "2.13.0",
+    "docpress": "0.6.12",
+    "eslint": "3.0.1",
```

Comment:
eslint 3 is too recent for us; the eslint configs need 2. We have to wait until they update to eslint 3.